integrations/huggingface-transformers/tutorials/token-classification/token-classification-cli.md

```diff
@@ -0,0 +1,273 @@
+# Token Classification: Sparse Transfer Learning with the CLI
+
+In this example, you will fine-tune a 90% pruned BERT model onto some token classification datasets using SparseML's CLI.
+
+### **Sparse Transfer Learning Overview**
+
+Sparse Transfer Learning is very similiar to the typical transfer learing process used to train NLP models, where we fine-tune a pretrained checkpoint onto a smaller downstream dataset. With Sparse Transfer Learning, we simply start the fine-tuning process from a pre-sparsified checkpoint and maintain sparsity while the training process occurs.
+
+### Pre-Sparsified BERT
+
+SparseZoo, Neural Magic's open source repository of pre-sparsified models, contains a 90% pruned version of BERT, which has been sparsified on the upstream Wikipedia and BookCorpus datasets with the masked language modeling objective. [Check out the model card](https://sparsezoo.neuralmagic.com/models/nlp%2Fmasked_language_modeling%2Fobert-base%2Fpytorch%2Fhuggingface%2Fwikipedia_bookcorpus%2Fpruned90-none). We will use this model as the starting point for the transfer learning process.
```

Comment:
We will use this model as the starting point for the transfer learning process. -> This model will be the starting point for the transfer learning process.